.github/workflows/tests.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     - name: Install RubyGems
       run: |
         gem install --user bundler
-        bundle install --retry=3 --jobs=4
+        bundle install --retry=3 --jobs=$(getconf _NPROCESSORS_ONLN)
```

Comment:
Actions VMs are not that powerful, so 4 is too many. So, specify the number of processors on the machine. This allows the same code to be used in the future even if the Actions VM becomes more powerful.

Reply:
Homebrew uses its own worker nodes.

Reply:
Not for this repo, just FYI.